### ISSUE TITLE
AttributeError: 'str' object has no attribute 'seek' When Running demo.py and yolo_v3.py

### ISSUE THREAD

**Issue by GuanzhaoLi** [mystic123/tensorflow-yolo-v3#9]
Hi mystic123!
Thank you for sharing your code with all of us! I just have a little problem when trying to run your code on sublime text3 on Windows 10. My error reports are posted as follow:
![q1](https://user-images.githubusercontent.com/40277753/41785788-45d45e3c-7609-11e8-857d-d20cba68e9b9.png)
![q2](https://user-images.githubusercontent.com/40277753/41785789-45e97538-7609-11e8-845b-9988447ed26d.png)

Could you kindly tell me how to solve them?

**Comment by mystic123** [comment]
Hi @GuanzhaoLi,
It look's like a problem with PIL (OSError: Cannot identify image file). Have you tried to search for a solution on stackoverflow?
I see that you use Windows. I have only tested it on Ubuntu and I don't have possibility to run this code on Windows.

**Comment by vmeschke** [comment]
@mystic123 I'm also having the same problem when running demo.py. Are we supposed to provide our own image file to run through the demo? If that's the case, some comments to indicate that in the demo may be helpful.

**Comment by mystic123** [comment]
The key elements of this repo are in yolo_v3.py, I added that tiny demo file just to check if it works.
In the blog post I published with this repo (https://medium.com/@pawekapica_31302/implementing-yolo-v3-in-tensorflow-tf-slim-c3c55ff59dbe), I noted that this tutorial and code are targeted to advanced TF devs which have some experience developing production-ready TF code (thus, understand how tf.app.flags work). I recommend just importing yolo_v3 architecture from yolo_v3.py and writing your own code for making detections. Demo.py is just a small hint how it may look like.

I added info how to run this demo in README.
